extension/src/popup/components/manageAssets/SelectAssetRows/index.tsx

```diff
@@ -23,20 +23,15 @@ import "./styles.scss";
 interface SelectAssetRowsProps {
   assetRows: ManageAssetCurrency[];
   maxHeight: number;
-  selectingAssetType: string;
```

Comment:
removing the selectingAssetType prop propagated through all the components anymore, much simpler 🕺